### PR TITLE
Fix python syntax of TFMA visualization

### PR DESCRIPTION
**Pull request by ajchili**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1972)

### CODE REVIEW

**Comment by ajchili** [comment]
TODO: Re-enable TFMA before merging (TFMA was disabled in #1986)

File: backend/src/apiserver/visualization/types/tfma.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 import tensorflow_model_analysis as tfma
 
-if variables.get("slicing_column", False) == False {
+if variables.get("slicing_column", False) == False:
```
[the remark this answers is not in the document]
Comment (IronPan) [reply backend/src/apiserver/visualization/types/tfma.py]: done

**Pull request merged**